### PR TITLE
update sig-docs, tech leads

### PR DESCRIPTION
**Pull request by kbhawkey**
Add @kbhawkey to tech leads.

### CODE REVIEW

**Comment by zacharysarah** [comment]
@kbhawkey became a SIG lead during the 03-Mar-2020 [SIG Docs weekly meeting](https://docs.google.com/document/d/1ddHwLK3kUMX1wVFIwlksjTk0MsqitBnWPe1LRa1Rx5A/edit). 

/lgtm

@jimangel @kbarnard10 Ready for your `/approve`.

**Comment by cblecker** [comment]
/approve
/hold

@jimangel @kbarnard10 @zacharysarah: Please remove hold when you're ready for this to merge.

**Comment by sftim** [comment]
@kbhawkey: is it OK to combine adding you & me into one PR?

**Comment by kbhawkey** [comment]
> 
> 
> @kbhawkey: is it OK to combine adding you & me into one PR?

Totally fine. Go for it.

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -40,6 +40,7 @@ aliases:
   sig-docs-leads:
     - jimangel
     - kbarnard10
+    - kbhawkey
```
Comment (sftim) [inline OWNERS_ALIASES]:
```suggestion
    - kbhawkey
    - sftim
```

File: sigs.yaml [hunk 2]
```diff
@@ -1240,6 +1240,10 @@ sigs:
     - github: zacharysarah
       name: Zach Corleissen
       company: Linux Foundation
+    tech_leads:
+    - github: kbhawkey
+      name: Karen Bradshaw
+      company: Independent
```
Comment (sftim) [inline sigs.yaml]:
```suggestion
    - github: kbhawkey
      name: Karen Bradshaw
      company: Independent
    - github: sftim
      name: Tim Bannister
      company: The Scale Factory
```

File: sig-docs/README.md [hunk 3]
```diff
@@ -32,6 +32,12 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * Kaitlyn Barnard (**[@kbarnard10](https://github.com/kbarnard10)**), Kong
 * Zach Corleissen (**[@zacharysarah](https://github.com/zacharysarah)**), Linux Foundation
 
+### Technical Leads
+The Technical Leads of the SIG establish new subprojects, decommission existing
+subprojects, and resolve cross-subproject technical issues and decisions.
+
+* Karen Bradshaw (**[@kbhawkey](https://github.com/kbhawkey)**), Independent
```
Comment (sftim) [inline sig-docs/README.md]:
```suggestion
* Karen Bradshaw (**[@kbhawkey](https://github.com/kbhawkey)**), Independent
* Tim Bannister (**[@sftim](https://github.com/sftim)**), The Scale Factory
```

**Comment by sftim** [comment]
:ok_hand: Thanks @kbhawkey

**Comment by jimangel** [comment]
/hold cancel
/lgtm

🎉 Congrats! SIG Docs is very lucky to have you both as leads!

**Pull request merged**